internal/constants/constants.go

```diff
@@ -12,6 +12,7 @@ const (
 	ClientPrivateKey   = "tls.key"
 	TrustedCABundleKey = "ca-bundle.crt"
 	Passphrase         = "passphrase"
+	BearerTokenFileKey = "bearer_token_file"
```

Comment:
Do we have precedence where we use this undscore ruby type syntax for keys instead of CamelCase?  Placing on hold pending comment from @alanconway

Reply:
I would call it just "token". 
- it is not a file, its the token data that's in the secret.
- "token" isn't likely  ambiguous in any case involving a bearer token.
- short, single words are less likely to be mis-spelled so fewer config errors.
Otherwise LGTM